574987574987--- u-boot.orig/configs/anbernic_rg35xx_h700_defconfig	2024-10-29 17:03:21.768276666 +0100
+++ u-boot/configs/anbernic_rg35xx_h700_defconfig	2024-10-29 17:17:21.719469834 +0100
@@ -25,3 +25,10 @@ CONFIG_REGULATOR_AXP=y
 CONFIG_AXP717_POWER=y
 CONFIG_AXP_DCDC2_VOLT=940
 CONFIG_AXP_DCDC3_VOLT=1100
+CONFIG_EFI_LOADER=n
+CONFIG_BOOTDELAY=0
+CONFIG_LED_STATUS=y
+CONFIG_LED_STATUS_GPIO=y
+CONFIG_LED_STATUS0=y
+CONFIG_LED_STATUS_BIT=267
+CONFIG_LED_STATUS_STATE=2
\ No newline at end of file